indico/modules/events/abstracts/blueprint.py

```diff
@@ -16,11 +16,12 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.events.abstracts.controllers.management import RHAbstracts
+from indico.modules.events.abstracts.controllers.management import RHAbstracts, RHAbstractList
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('abstracts', __name__, url_prefix='/event/<confId>', template_folder='templates',
                       virtual_template_folder='events/abstracts')
 
 # Management
-_bp.add_url_rule('/manage/abstracts/', 'manage_abstracts', RHAbstracts)
+_bp.add_url_rule('/manage/abstracts/setup', 'manage_abstracts', RHAbstracts)
```

Comment:
I wanted to use |/manage/abstracts/ |for the list since we use the same 
for contributions and registrations.